onnxruntime/test/util/test_utils.cc

```diff
@@ -117,6 +117,57 @@ void RunAndVerifyOutputsWithEP(const ORTCHAR_T* model_path, const char* log_id,
   VerifyOutputs(output_names, expected_fetches, fetches);
 }
 
+void RunAndVerifyOutputsWithEPViaModelData(std::string model_data, const char* log_id,
```

Comment:
```
void RunAndVerifyOutputsWithEPViaModelData(const std::string& model_data, ...
```

Reply:
> ```
> void RunAndVerifyOutputsWithEPViaModelData(const std::string& model_data, ...
> ```

sorry should have used const lvalue here...